e2etest/utils.go

```diff
@@ -50,11 +50,21 @@ func wgetRetry(address string, exc executor.Executor) error {
 	return err
 }
 
-func tweakServicePort(svc *v1.Service) {
+func tweakService(svc *v1.Service, ipFamily string) {
 	if servicePodPort != 80 {
 		// if servicePodPort is non default, then change service spec.
 		svc.Spec.Ports[0].TargetPort = intstr.FromInt(int(servicePodPort))
 	}
+	if ipFamily == "ipv6" {
+		policy := v1.IPFamilyPolicyPreferDualStack
+		svc.Spec.IPFamilies = []v1.IPFamily{v1.IPv6Protocol, v1.IPv4Protocol}
```

Comment:
Also, by doing this we'd have duplicate tests in the dual stack lane, because we are asking for both families in the v4 and in the v6 case.